_ext/pipeline.rb

```diff
@@ -137,7 +137,8 @@
+                                                          excludes: ['_fuse-quickstarts/template','_fuse-quickstarts/switchyard','_fuse-quickstarts/switchyard/bean-service','_fuse-quickstarts/switchyard/bpel-jms-binding','_fuse-quickstarts/switchyard/bpel-loan-approval','_fuse-quickstarts/switchyard/bpel-say-hello','_fuse-quickstarts/switchyard/bpel-simple-correlation','_fuse-quickstarts/switchyard/camel-amqp-binding','_fuse-quickstarts/switchyard/camel-atom-binding','_fuse-quickstarts/switchyard/camel-bindy','_fuse-quickstarts/switchyard/camel-bus-cdi','_fuse-quickstarts/switchyard/camel-cxf-binding','_fuse-quickstarts/switchyard/camel-file-binding','_fuse-quickstarts/switchyard/camel-ftp-binding','_fuse-quickstarts/switchyard/camel-hl7','_fuse-quickstarts/switchyard/camel-jaxb','_fuse-quickstarts/switchyard/camel-mail-binding','_fuse-quickstarts/switchyard/camel-mqtt-binding','_fuse-quickstarts/switchyard/camel-netty-binding','_fuse-quickstarts/switchyard/camel-quartz-binding','_fuse-quickstarts/switchyard/camel-rss-binding','_fuse-quickstarts/switchyard/camel-sap-binding','_fuse-quickstarts/switchyard/camel-saxon','_fuse-quickstarts/switchyard/camel-service','_fuse-quickstarts/switchyard/camel-soap-proxy','_fuse-quickstarts/switchyard/camel-sql-binding','_fuse-quickstarts/switchyard/demos/multiApp','_fuse-quickstarts/switchyard/demos/multiApp/artifacts','_fuse-quickstarts/switchyard/demos/multiApp/order-consumer','_fuse-quickstarts/switchyard/demos/multiApp/order-service','_fuse-quickstarts/switchyard/demos/multiApp/web','_fuse-quickstarts/switchyard/demos/orders','_fuse-quickstarts/switchyard/demos/policy-security-basic','_fuse-quickstarts/switchyard/demos/policy-security-cert','_fuse-quickstarts/switchyard/demos/policy-security-saml','_fuse-quickstarts/switchyard/demos/policy-security-wss-signencrypt','_fuse-quickstarts/switchyard/demos/policy-security-wss-username','_fuse-quickstarts/switchyard/demos/policy-transaction','_fuse-quickstarts/switchyard/demos/security-propagation','_fuse-quickstarts/switchyard/demos/transaction-propagation','_fuse-quickstarts/switchyard/demos/webapp-deploy','_fuse-quickstarts/switchyard/ear-deployment','_fuse-quickstarts/switchyard/http-binding','_fuse-quickstarts/switchyard/jca-inflow-activemq','_fuse-quickstarts/switchyard/jca-inflow-hornetq','_fuse-quickstarts/switchyard/jca-outbound-activemq','_fuse-quickstarts/switchyard/jca-outbound-hornetq','_fuse-quickstarts/switchyard/remote-invoker','_fuse-quickstarts/switchyard/rest-binding','_fuse-quickstarts/switchyard/soap-attachment','_fuse-quickstarts/switchyard/soap-binding-rpc','_fuse-quickstarts/switchyard/soap-mtom','_fuse-quickstarts/switchyard/transform-dozer','_fuse-quickstarts/switchyard/transform-jaxb','_fuse-quickstarts/switchyard/transform-json','_fuse-quickstarts/switchyard/transform-smooks','_fuse-quickstarts/switchyard/transform-xslt','_fuse-quickstarts/switchyard/validate-xml'],
```

Comment:
Can a regex be used here? This would save you from having to list every dir under 'switchyard' which may change in the future.